docs/guides/pagination.mdx

```diff
@@ -0,0 +1,234 @@
+---
+id: api-rest-pagination
+title: API REST Pagination
+sidebar_label: Pagination
+---
+
+Pagination is a strategy that limits access to large datasets by dividing the data into manageable chunks or "pages." This
+approach provides mechanisms for accessing subsequent or previous pages of data.
+
+When dealing with API responses that could potentially return large datasets, pagination becomes essential for two main reasons:
+
+1. **Mitigate data access abuse**: Pagination limits the amount of data that can be accessed in a single request. Clients or
+   consumers can only retrieve a defined subset of data per page, preventing potential system overload or abuse.
+
+2. **Improve response performance**: By returning smaller sets of data, pagination ensures faster response times. This is
+   particularly beneficial when dealing with large datasets, as it reduces the load on both the server and the client.
+
+Ory supports two pagination strategies:
+
+- **Token-based pagination**: The primary and recommended method for paginating through data in Ory's API.
+- **Offset pagination**: A deprecated method that is still supported but not recommended for new implementations.
+
+In the following sections, we'll explore each of these pagination strategies in detail, with a focus on the recommended
+token-based approach.
+
+## Token pagination
+
+Ory implements token-based pagination as the primary method for handling large datasets in list operations. This approach offers
+better performance and consistency compared to traditional offset-based pagination.
+
+![Token pagination illustration](./_static/pagination/token-pagination.png)
+
+### Default behavior and parameters
+
+By default, Ory paginates list data with a page size of 250 items per page. However, you can customize this behavior using two key
```

Comment:
The page size is typically dependent on the API and may also change in the future (ie get smaller or larger). There is usually a max page size

Reply:
Interesting. I couldn't find a max page size value in both the response headers and body of admin/identities. Should I be looking somewhere else?

Reply:
It's only documented in the code at the moment. It's here: https://github.com/ory/kratos/blob/23f323274616c7383668ecb3c405416b8a0e30f0/identity/handler.go#L202